### PR TITLE
v1.7: Add Hugging Face vector search updates

### LINKED ISSUE #2736: v1.7: Huggingface embedder updates
v1.7 allows users to generate embeddings with NVidia GPUs.

## Questions
- Is this being exposed on Meilisearch Cloud? If so, will it require any user input or will it be sorted out on our end?

## Tasks
- [ ] Update `learn/experimental/vector_search`

## References
- SME: @dureuill 
- Initial documentation: https://github.com/meilisearch/meilisearch/issues/4306#issuecomment-1893849620
- Engine issue: https://github.com/meilisearch/meilisearch/issues/4306

### PR DESCRIPTION
**Pull request by guimachiavelli**
Closes #2736 

@Strift and @CaroFG, notifying you this PR includes a short guide aimed at enabling GPU usage with the Hugging Face embedding. It's aimed at advanced self-hosted users, but thought you might want to be aware of it.

### CODE REVIEW

File: learn/cookbooks/computing_hugging_face_embeddings_gpu.mdx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+After you have installed CUDA in your machine, run the following command in your command-line terminal:
+
+```sh
+nvcc --version | head -1
+```
+
+If CUDA is working correctly, you will see the following response:
+
+```
+nvcc: NVIDIA (R) Cuda compiler driver
```
Comment (dureuill) [inline learn/cookbooks/computing_hugging_face_embeddings_gpu.mdx:29]: I think this one might be cut short, was it not?

**Review by dureuill — APPROVED** [review]
:+1:

**Comment by Strift** [comment]
@guimachiavelli the rest of docs isn't enough for you anymore you have to write in our cookbooks now 😒

(thanks for the ping)

File: learn/cookbooks/computing_hugging_face_embeddings_gpu.mdx (hunk 1 above)
Comment (guimachiavelli) [reply learn/cookbooks/computing_hugging_face_embeddings_gpu.mdx:29]: The code sample pipes the result into `head -1` so we only get the first line. As far as I understand, we only care about receiving a positive reply here stating cuda was correctly installed, so I thought using `head` would be simpler and more to the point.

I can change it if you think it's better to have the full output, though.

Comment (dureuill) [reply learn/cookbooks/computing_hugging_face_embeddings_gpu.mdx:29]: OK I didn't see the head -1, that makes sense then

It doesn't need to be changed

**Pull request merged**